src/components/NotificationButtons/NotificationButtons.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
remove this component

Reply:
can I remove it after merging (or creating pull request) another component combined with notifications?